src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/Infra/ControlTestBase.cs

```diff
@@ -14,6 +14,7 @@
 namespace System.Windows.Forms.UITests;
 
 [UseDefaultXunitCulture]
+[UISettings(MaxAttempts = 3)] // Retry 3 times before failing.
```

Comment:
Consider either of these:

```suggestion
[UISettings(MaxAttempts = 3)] // Try up to 3 times before failing.
```

```suggestion
[UISettings(MaxAttempts = 3)] // Retry 2 times before failing.
```